### PR TITLE
Possibility to search only folders

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Needed for https://github.com/nextcloud/android/pull/1930

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: src/com/owncloud/android/lib/resources/files/SearchOperation.java [hunk 1]
```diff
@@ -66,10 +66,12 @@
 
     private String searchQuery;
     private SearchType searchType;
+    private boolean onlyFolder;
 
-    public SearchOperation(String query, SearchType searchType) {
+    public SearchOperation(String query, SearchType searchType, boolean onlyFolder) {
```
Comment (mario) [inline src/com/owncloud/android/lib/resources/files/SearchOperation.java]: Rename to filterOutFiles please.

**Comment by mario** [comment]
One small comment, otherwise 👍

**Comment by mario** [comment]
👍

**Comment by tobiasKaminsky** [comment]
Merging with only one approval as @AndyScherzinger is on vacation.

**Pull request merged**